source/_integrations/switchbot.markdown

```diff
@@ -64,6 +64,7 @@ Some SwitchBot devices need to be configured within the app before being control
 - Plug Mini (WoPlug), both the original (model W1901400) and HomeKit-enabled (model W1901401)
 - [Lock (WoLock)](https://switch-bot.com/pages/switchbot-lock) (currently the Pro model is not supported)
 - [Blind Tilt (WoBlindTilt)](https://switch-bot.com/pages/switchbot-blind-tilt)
+- [Hub 2 (WoHub2)](https://switch-bot.com/pages/switchbot-hub-2)
```

Comment:
Maybe add a comment that it only supports the sensors since users may expect every device the hub supports is now supported which isn’t the case

Reply:
Thank you, I have made the changes.